src/pages/sign-in-page/sign-in-page.tsx

```diff
@@ -29,7 +31,17 @@ export const SignInPage = () => {
   );
 
   function handleSubmitClick() {
-    console.log(userName, password);
+    userClient.login(userName, password).then((success) => {
```

Comment:
Is there a reason here for using `.then` instead of `await`?

Reply:
no, making the function async and awaiting this is equivalent
but i'll change it to be consistent w our other async handlers